eodhp-web-presence/templates/service-account.yaml

```diff
@@ -1,9 +1,9 @@
-{{ if .Values.serviceAccount.create }}
+{{ if .Values.web.serviceAccount.create }}
 apiVersion: v1
 automountServiceAccountToken: false
 kind: ServiceAccount
 metadata:
-  name: {{ .Values.serviceAccount.name }}
+  name: {{ .Values.web.serviceAccount.name }}
   annotations:
-    eks.amazonaws.com/role-arn: {{ .Values.serviceAccount.annotations.roleArn }}
+    eks.amazonaws.com/role-arn: {{ .Values.web.serviceAccount.annotations.roleArn }}
```

Comment:
don't hard code this annotation key, use `{{- toYaml .Values.web.serviceAccount.annotations | nindent XX }}` (similar to how you include env vars) to make it more flexible.

Reply:
Updated